backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/SysMLElementSerializer.java

```diff
@@ -385,8 +386,20 @@ public String caseOperatorExpression(OperatorExpression op) {
 
     @Override
     public String caseInvocationExpression(InvocationExpression expression) {
-        LOGGER.warn("InvocationExpression are not handled yet");
-        return "";
+        Appender builder = this.newAppender();
+        List<Relationship> relationships = expression.getOwnedRelationship();
+        if (!relationships.isEmpty() && (relationships.get(0) instanceof FeatureTyping || relationships.get(0) instanceof Subsetting)) {
+            relationships.stream()
+                .filter(Specialization.class::isInstance)
+                .map(Specialization.class::cast)
+                .filter(specialization -> specialization instanceof FeatureTyping || specialization instanceof Subsetting)
```

Comment:
You already check in the _if_ statement that the `relationships ` contains a _FeatureTyping_ or a _Subsetting_ in first position.
So I think this `.filter(...)` could be removed.

Reply:
done